src/stories/pages/Experiment/Experiment.js

```diff
@@ -41,106 +51,55 @@ export const Experiment = ({ title }) => {
         <Constrain>
           <Wizard labels={steps}>
             <div className="step-1">
-              <h3>Introduction</h3>
-              <h4>{title}</h4>
-              <p>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens
-                mnesarchum, ei periculis adipiscing per, probo populo nec ad.
-              </p>
-              <p>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens
-                mnesarchum, ei periculis adipiscing per, probo populo nec ad.
-              </p>
+              <Section titleEl={HEADING} title='Introduction'>
+                <h4>{title}</h4>
+                {description}
+              </Section>
             </div>
             <div className="step-2">
-              <h3>Consent Form</h3>
-              <p>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens
-                mnesarchum, ei periculis adipiscing per, probo populo nec ad.
-              </p>
-              <p>
-                Lorem ipsum dolor sit amet, an has summo
-                riure epicuri, has illud rationibus et. Prima ridens sit te,
-                nam idque explicari expetendis in. An mei adolescens
-                mnesarchum, ei periculis adipiscing per, probo populo nec ad.
-              </p>
+              <Section titleEl={HEADING} title='Consent Form'>
+                <p>Please, complete the form below before completing the
+                  study.</p>
+                { consentForms.map((form, i) => {
```

Comment:
Just have a question here.

Does this mean there are multiple consent forms as this is a map?

Reply:
Yes!